contracts/ThalesRoyale/ThalesRoyale.sol

```diff
@@ -232,9 +245,56 @@ contract ThalesRoyale is Owned, Pausable {
         rounds = _rounds;
     }
 
+
+    function setClaimTime(uint _claimTime) public onlyOwner {
+        claimTime = _claimTime;
+    }
+
+    function _populateWinnersAndThereRewards(address[] memory alivePlayers) internal {
+        require(finished, "Royale must be finished");
+        require(alivePlayers.length > 0, "There is no alive players left in Royale");
+
+        uint rewardPerPlayer = reward.divideDecimal(alivePlayers.length);
```

Comment:
what if 10000 TH / 3 alive players = 3333.33333333.... 👃

is this we need to use 🤔

Reply:
done.